static/js/components/dashboard/courses/StatusMessages.js

```diff
@@ -157,6 +157,15 @@ export const calculateMessages = (props: CalculateMessagesProps) => {
           >
             View Certificate
           </a>
+          {S.maybe(
+            null,
+            () => (
+              <a onClick={() => setShowExpandedCourseStatus(course.id)}>
+                {" | Re-enroll"}
```

Comment:
Could you separate the vertical bar and the link? when I hover over it it looks broken.